scripts/cloud-api-demo/check_permissions.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
```

Comment:
Add comments regarding the purpose of this script file

Reply:
Added